source/Calamari.Common/Features/Behaviours/ConfiguredScriptBehaviour.cs

```diff
@@ -14,7 +15,7 @@
 
 namespace Calamari.Common.Features.Behaviours
 {
-    public class ConfiguredScriptBehaviour : IBehaviour
+    public abstract class ConfiguredScriptBehaviour : IBehaviour
```

Comment:
I don't think we need these changes?

Reply:
Maybe the `abstract` is not required. I created subclasses for each of the deployment stages to remove the DI resolve with named parameters.